atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/KeyValueServicePuncherStore.java

```diff
@@ -43,7 +52,24 @@
 /**
  * A PuncherStore implemented as a table in the KeyValueService.
  *
- * @author jweel
+ * This utilises dynamic columns to remove the necessity of range scans.
+ *
+ * The schema is:
+ *
+ * row: time rounded to 14 days
```

Comment:
nit: `truncated`?

Reply:
not quite truncated, since we always round down